spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -0,0 +1,123 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[release_notes_10]]
+== Release Notes for MicroProfile LRA 1.0
+
+http://download.eclipse.org/microprofile/microprofile-lra-1.0/microprofile-lra.pdf[MicroProfile LRA Spec PDF]
+http://download.eclipse.org/microprofile/microprofile-lra-1.0/microprofile-lra.html[MicroProfile LRA Spec HTML]
+http://download.eclipse.org/microprofile/microprofile-lra-1.0/apidocs/[MicroProfile Propagation LRA Javadocs]
+
+Key features:
+
+A transaction model which isn’t full ACID:
+
+- an activity reflects business interactions
+- all scoped work must be compensatable
+- activities are visible to other services
+- when an activity ends all work is either accepted or all work is compensated
+- the LRA model defines the triggers for when and where compensation actions are executed
+- defines annotations for the safe/transactional execution of activities supporting long running activities involving loosely coupled processes
+
+Supports:
+
+- relaxion of atomicity (using nested transactions);
+- locking is optional  (=> loss of isolation);
+- forward progress by allowing work to finish early, to provisionally perform subsets of work (nesting), time bounds, composition of activities
+
+Provides CDI annotations:
+
+.LRA Annotations
+|===
+|Annotation |Description |JAX-RS
+
+|@LRA
+|Controls the life cycle of an LRA
+|Yes
+
+|@AfterLRA
+|Notification that an LRA has finished
+|Yes/Optional
+|===
+
+.Participant Annotations
+|===
+|Annotation |Description |JAX-RS
+
+|@Compensate
+|Indicates that the method should be invoked if the LRA is cancelled.
+|Optional
+
+|@Complete
+|Indicates that the method should be invoked if the LRA is closed.
+|Optional
+|@Leave
+|Indicates that this class is no longer interested in this LRA.
+|Yes
+|@Status
+|When the annotated method is invoked it should report the status.
+|Optional
+|@Forget
+|The method may release any resources associated with the LRA
+|Optional
+|===
+
+* reactive support:
+** CompletionStage
+** @Suspended AsyncResponse
+** HTTP 202 Accepted response code
+
+To get started, add this dependency to your project:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.eclipse.microprofile.lra</groupId>
+    <artifactId>microprofile-lra-api</artifactId>
+    <version>${version.microprofile.lra}</version>
+    <version>1.0</version>
+    <scope>provided</scope>
+</dependency>
+----
+
+Create a JAX-RS business resource and annotate the methods that you would like to be included in a long running action
+using the @LRA annotation. Mimally you should define which business method that should be run if the LRA is cancelled
+using the @Compensate annotation.
+
+[source,java]
+----
+public class SimpleParticipant {
```

Comment:
Not necessary, but we can add Path annotation also on class, just to make it clear this is a JAX-RS resource. Up to you if you want to update it.